packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/PinnedColumnsTable.jsx

```diff
@@ -54,6 +54,7 @@ const PinnedColumnsTable = () => (
     pinnedColumns={tableData.cols.slice(0, 2)}
     overflowColumns={tableData.cols.slice(2)}
     rows={tableData.rows}
+    ariaLabel="Table"
```

Comment:
Shouldn't our aria labels describe the example.  They are the same value for every example.